pages/en/lb3/3.0-Release-Notes.md

```diff
@@ -30,6 +30,8 @@ dependency.
 
 See [loopback#275](https://github.com/strongloop/loopback/issues/275).
 
+Please see [how to migrate](http://loopback.io/doc/en/lb3/Migrating-to-3.0.html#loopback-datasource-juggler-is-a-regular-dependency-now)
```

Comment:
Remove Please for consistency with the line above? Or add please to the other ones too. Ditto for the rest of your changes below.